client/src/blog/post.tsx

```diff
@@ -106,6 +107,39 @@ function BlogImageFigure({
   );
 }
 
+function PreviousNext({
+  metadata: { previous, next },
+}: {
+  metadata: BlogPostFrontmatter;
+}) {
+  return (
+    <section className="previous-next">
+      {previous && (
+        <PreviousNextLink direction="Previous" metadata={previous} />
+      )}
+      {next && <PreviousNextLink direction="Next" metadata={next} />}
+    </section>
+  );
+}
+
+function PreviousNextLink({
+  direction,
+  metadata: { slug, title },
+}: {
+  direction: string;
+  metadata: BlogPostLimitedFrontmatter;
+}) {
+  return (
+    <a href={`/en-US/blog/${slug}/`} className={direction.toLowerCase()}>
+      <article>
+        <h1>
+          <strong>{direction} Post</strong> {title}
+        </h1>
```

Comment:
🟠 Afaik we shouldn't have two `<h1>`s in a document for a11y reasons.

Reply:
I thought it was acceptable in separate `<article>`s - alas, no: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page